### ISSUE TITLE
predicted-result will be different for same video input and model

### ISSUE THREAD

**Issue by xhappy** [qijiezhao/pseudo-3d-pytorch#6]
Hi Qijie,

I just found the computed probability will  be changed for same input and model. For example, for the video 12065806-102-009-013814.mp4, I predicted this video using your codes (have added softmax layer) and the predicted result will be different if run the command "python3 p3d_model.py 12065806-102-009-013814.mp4" multiple times

In my view, same video input, same model  and same weights, same codes, It must be same predicted result. Could you please help to analysis or fix it? Thanks in advance.

**Comment by qijiezhao** [comment]
Predict a video using p3d is non-trivial, If you want to get the same result as the author's, you should first implement the same sampling steps and pre-processing steps. Detailly, 1) evenly sample continuous 16-frames from the given video. 2)pre-process these frames together, including 'resize' and 'normalize'. 3) average the results of the tested clips(each clip is 16-frames long). 
Alternatively, you can sample like you want, you can also compute the final result with linear weights of all the results.  If you fix these, I believe the result will not vary anymore.
